db/migrate/20200220161956_add_equality_and_diversity_to_application_form.rb

```diff
@@ -0,0 +1,5 @@
+class AddEqualityAndDiversityToApplicationForm < ActiveRecord::Migration[6.0]
+  def change
+    add_column :application_forms, :equality_and_diversity, :jsonb
+  end
+end
```

Comment:
Needs a rebase since https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1446 is already merged with this?

Reply:
This is still here? 🤔